<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformerPage</class>
 <widget class="QWidget" name="InformerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информатор</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="14" column="0">
    <widget class="QCheckBox" name="chkInformerShowByUserArea">
     <property name="text">
      <string>Фильтровать уведомления по участку пользователя</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <widget class="QCheckBox" name="chkInformerShowPersonSNILS">
     <property name="text">
      <string>Фильтровать уведомления по СНИЛСу пользователя</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QCheckBox" name="chkInformerShowNoSNILS">
     <property name="text">
      <string>Показывать уведомления без СНИЛС</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QCheckBox" name="chkInformerShowByUserNotArea">
     <property name="text">
      <string>Фильтровать уведомления по пациентам без участка</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkInformerShowPersonSNILS</tabstop>
  <tabstop>chkInformerShowNoSNILS</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
